bin/exempt-lint.mjs

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+/**
+ * @file
+ * You can use this script to exempt an app file from a specific eslint rule.
+ *
+ * This should only be used to exempt existing apps when a new lint rule is added.
+ * You are not allowed to exempt your new app from existing lint rules.
+ *
+ * Run it like this:
+ * node bin/exempt-lint.mjs LINTRULE FILEPATH
+ *
+ * Example command:
+ * node bin/exempt-lint.mjs no-unused-vars ./apps/_example_app/app.js
+ */
+
+import fs from "node:fs/promises";
+
+const lintRule = process.argv[2];
```

Comment:
Same here (and with `sync-lint-exemptions.mjs`) as above with dealing with all arguments :)